src/client/components/Pages.jsx

```diff
@@ -1,10 +1,74 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
 
-const Pages = () => (
-  <div id="pages">
-    <button type="submit">Page 1</button>
-    <button type="submit">Page N</button>
-  </div>
-);
+const Container = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-top: 1rem;
+  padding-bottom: 1rem;
+  font-family: BrandonText;
+`;
+
+const InnerContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+`;
+
+const PageButton = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 2rem;
+  height: 2rem;
+  margin-left: 0.4rem;
+  margin-right: 0.4rem;
+  border-radius: 50%;
+  border: solid 1px;
+  border-color: ${(props) => (props.active ? 'red' : '#d8d9db')};
+  color: #2d333f;
+  cursor: pointer;
+`;
+
+const Pages = (props) => {
+  const { pageCount, activePage, filterPage } = props;
+  const changePage = (event) => {
+    const page = Number(event.target.dataset.value);
+    filterPage(page);
+  };
+  const pageButtons = [];
+  for (let i = 1; i <= pageCount; i += 1) {
+    const active = i === activePage;
+    pageButtons.push(
+      <PageButton key={i} data-value={i} active={active} onClick={changePage}>
```

Comment:
Instead of passing data-value as a custom attribute and relying on DOM nodes and events for your functions, couldn't you just change your function to pass the value of i directly to make both the element and function look cleaner?

Reply:
The rationale there was for the left and right arrows - they don't technically have an i value, it has to be calculated based on the current active page, and varies depending on whether they're at the beginning, middle, or end of the list. Good observation though - I'll give this some more thought to see if it can be cleaned up.